en/appbuilder/using-data-in-your-app.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Indigo Design App Builder Software overview
+title: Using data in the Indigo Design App Builder
 _description: Indigo Design App Builder enables users to add external data sources and bind them to the applications designed within Indigo Design App Builder
 _keywords: App builder, Indigo Design, Infragistics, Data Sources, Data Binding
 ---
 
 # Data Feature overview 
-The Indigo Design App Builder Data features lets users add, edit and use external data source in the application they are developing. By default, the Indigo Design App Builder comes with a mock Northwind data source which users can use for reference. Apart from it, the data source types that can be added are JSON (Rest API), Excel or CSV files. All added data sources are placed in the Data Toolbox and users can expand/collapse each data source in order to see the included tables and selected fields.
+The Indigo Design App Builder Data features lets users add, edit and use external data source in the application they are developing. By default, the Indigo Design App Builder comes with a mock Northwind data source which users can use for reference. Currently, a Rest API data source types can be added, enabling users to upload a JSON file or provide a URL with the Data Source. All added data sources are placed in the Data Toolbox and users can expand/collapse each data source in order to see the included tables and selected fields.
```

Comment:
Same comment as in the Changelog section.

Reply:
I will leave only Northwind, Rest API, JSON capitalized since they are names.

Reply:
@DaniMarinov I meant that "uploading a JSON file" is not a REST API data source.

Reply:
It should be ok now @kdinev